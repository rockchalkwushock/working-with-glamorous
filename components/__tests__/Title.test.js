import { shallow, render, mount } from 'enzyme'

import { Title } from '../'

describe('Title Component', () => {
  let ui
  test('should render to the DOM', () => {
    ui = <Title />
    expect(shallow(ui)).toMatchSnapshotWithGlamor(`enzyme.shallow`)
    expect(mount(ui)).toMatchSnapshotWithGlamor(`enzyme.mount`)
    expect(render(ui)).toMatchSnapshotWithGlamor(`enzyme.render`)
  })
})
