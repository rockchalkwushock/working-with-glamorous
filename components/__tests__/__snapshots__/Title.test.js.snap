// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enzyme.mount 1`] = `
.css-1oawyek,
[data-css-1oawyek] {
  color: #FEBA00;
  font-family: "Pacifico", cursive;
  margin: 0px;
  padding: 1em 0 1em 0;
}

<Title>
  <StyledTitle>
    <h1
      className="css-1oawyek"
    >
      Glamorous SearchBar
    </h1>
  </StyledTitle>
</Title>
`;

exports[`enzyme.render 1`] = `
.css-1oawyek,
[data-css-1oawyek] {
  color: #FEBA00;
  font-family: "Pacifico", cursive;
  margin: 0px;
  padding: 1em 0 1em 0;
}

<h1
  class="css-1oawyek"
>
  Glamorous SearchBar
</h1>
`;

exports[`enzyme.shallow 1`] = `
<StyledTitle>
  Glamorous SearchBar
</StyledTitle>
`;
